otri/filtering/filters/merge_filter.py

```diff
@@ -1,41 +1,59 @@
-from ..filter import Filter, Stream, Collection
+from ..filter import Filter, Stream, Sequence, Mapping, Any
 
 
 class SequentialMergeFilter(Filter):
     '''
-    Merges elements from multiple streams into one single output.
+    Sequentially merges elements from multiple streams into one single output.
+    
     Input:
         Multiple streams.
     Output:
         A single stream containing data read sequentially (all of stream 1, then all of stream 2 and so on).
     '''
 
-    def __init__(self, input_streams: Collection[Stream]):
+    def __init__(self, input: Sequence[str], output: str):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)